testsuite/features/secondary/min_ssh_push.feature

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2020 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Register a salt system to be managed via SSH push
```

Comment:
I would prefer to use the same naming everywhere, not sure which is the better one, I would say "via SSH Tunnel" ?